internal_transfers/actions/tests/parse.spec.ts

```diff
@@ -1,33 +1,34 @@
-import {
-  partitionEventLogs,
-  SETTLEMENT_CONTRACT_ADDRESS,
-  SETTLEMENT_EVENT_TOPIC,
-  TRADE_EVENT_TOPIC,
-  TRANSFER_EVENT_TOPIC,
-} from "../src/parse";
+import { partitionEventLogs } from "../src/parse";
+// @ts-ignore
```

Comment:
This seems weird. Out of curiosity what are we ignoring here?

Reply:
I am not sure. I think my IDE may have snuck this in here. Gotta check.